app/consapp/convbin/gcc/makefile

```diff
@@ -149,21 +139,21 @@ test2: # sbp json with spaces
 	./sbp2rinex $(DATDIR)$(SEP)2017-05-12-v1.1.26.sbp.json -o 2017-05-12-v1.1.26-json.obs -n 2017-05-12-v1.1.26-json.nav
 test3: test1 test2 # ensure outputs from test1 and test2 are bit exact minus first two lines
 	@echo -e "\n\n****** Test3: compare json and binary output ********\n"
-	tail -n +5 ./2017-05-12-v1.1.26-json.obs > 2017-05-12-v1.1.26-json.trunc.obs
-	tail -n +5 ./2017-05-12-v1.1.26-bin.obs > 2017-05-12-v1.1.26-bin.trunc.obs
+	tail -n +8 ./2017-05-12-v1.1.26-json.obs > 2017-05-12-v1.1.26-json.trunc.obs
+	tail -n +8 ./2017-05-12-v1.1.26-bin.obs > 2017-05-12-v1.1.26-bin.trunc.obs
 	diff -q 2017-05-12-v1.1.26-json.trunc.obs 2017-05-12-v1.1.26-bin.trunc.obs
 test4:
 	@echo -e "\n\n***** Test: json without spaces********\n"
 	./sbp2rinex $(DATDIR)$(SEP)2017-05-12-v1.1.26-nospaces.sbp.json -o 2017-05-12-v1.1.26-json-nospaces.obs -n 2017-05-12-v1.1.26-json.nav
 test5: test1 test4  # make sure JSON with no spaces are bit exact minus first two lines
-	tail -n +5 ./2017-05-12-v1.1.26-json-nospaces.obs > 2017-05-12-v1.1.26-json-nospaces.trunc.obs
+	tail -n +8 ./2017-05-12-v1.1.26-json-nospaces.obs > 2017-05-12-v1.1.26-json-nospaces.trunc.obs
 	diff 2017-05-12-v1.1.26-json-nospaces.trunc.obs 2017-05-12-v1.1.26-bin.trunc.obs
 test6: # BDS + GAL + GLO + GPS
 	@echo -e "\n\n****** Test6: BDS + GAL + GLO + GPS ********\n"
 	./sbp2rinex $(DATDIR)$(SEP)2018-06-15-v1.4.0-develop-2018061321-3-g734bd4b9.sbp.json -o 2018-06-15-json.obs -n 2018-06-15-json.nav
-	tail -n +4 ./2018-06-15-json.obs > 2018-06-15-json.trunc.obs
-	diff --strip-trailing-cr ./2018-06-15-json.trunc.obs $(shell pwd)$(SEP)$(DATDIR)$(SEP)2018-06-15-json.obs
-	tail -n +4 ./2018-06-15-json.nav > 2018-06-15-json.trunc.nav
-	diff --strip-trailing-cr ./2018-06-15-json.trunc.nav $(shell pwd)$(SEP)$(DATDIR)$(SEP)2018-06-15-json.nav
+	tail -n +8 ./2018-06-15-json.obs > 2018-06-15-json.trunc.obs
+	#diff --strip-trailing-cr ./2018-06-15-json.trunc.obs $(shell pwd)$(SEP)$(DATDIR)$(SEP)2018-06-15-json.obs
```

Comment:
Are we able to re-enable thse tests now?

Reply:
with the latest commit, i got the following difference for observation file, and for navigation file. The observation file generated from this PR has a more meaningful header, and the navigation file has one more ephemeris for GPS, and one orbital element that is more up to date. So i would use the generated observation file and navigation from this PR as a new reference.  
<img width="1757" alt="Screen Shot 2022-02-08 at 9 35 18 pm" src="https://user-images.githubusercontent.com/19918984/152971507-0141f307-9526-4aa1-81d0-3ce20df8f6a4.png"
<img width="1782" alt="Screen Shot 2022-02-08 at 9 35 56 pm" src="https://user-images.githubusercontent.com/19918984/152971704-19bf502f-6dcc-489f-86b9-e3ab2fceea74.png">
>